docs/ballerina-by-example/examples/sql-connector/sql-connector.bal

```diff
@@ -3,16 +3,16 @@ import ballerina.io;
 
 function main (string[] args) {
     endpoint<sql:ClientConnector> testDB {
-          create sql:ClientConnector(sql:DB.MYSQL, "localhost", 3306,
-            "testdb", "root", "root", {maximumPoolSize:5});
+          create sql:ClientConnector(
+          sql:DB.MYSQL, "localhost", 3306, "testdb", "root", "root", {maximumPoolSize:5});
     }
-    //Create a DB table using update action.If the DDL
-    //statement execution is success update action returns 0.
+    //Create a DB table using updateQuery action.If the DDL
```

Comment:
Revert all updateQuery to update. Check the all places

Reply:
fixed.